instructure/canvas-ios
# assignment list using graphql with grading periods and assignment groups

refs: none
affects: Student
release note: none

Test Plan
- run through some courses and view assignments. 
- Make sure you can filter by grading period and clear the filter.  
- make sure paging of assignments is working.
